### ISSUE TITLE
bug: SCSS loading rules inconsistent with CSS loading

### ISSUE THREAD

**Issue by bostrom** [driftyco/ionic#5492]
**Type**: <span ionic-type>bug</span>

**Ionic Version**: <span ionic-version>1.x</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>There seems to be some inconsistencies with the bundled ionic CSS file and the SCSS file. When generating CSS from the SCSS, at least the loading popup is no longer visible. Glancing at the differences between these two, there seems to be more than just the loading modal.</span>

<span is-issue-template></span>

**Comment by jgw96** [comment]
Thanks for submitting an issue with us! Would you mind sharing a code sample with us so that we can better investigate the issue you are having? Thanks again!

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! Since it has been a little while since there was any activity on this issue i will be closing it, but feel free to reopen this issue if needed!

**Comment by bostrom** [comment]
@jgw96 Hi and thanks for paying attention to my bug report. Sorry for not responding, I've had some busy weeks. 

This issue has not likely disappeared, so I'd leave it open. I'll try to find some time to provide you with more precise information about where the SCSS<->CSS mismatches are within the coming weeks.

**Comment by bostrom** [comment]
@jgw96 Allright, I've got some more info on this one now. 

So the app I've generated uses the $iconicLoading service for displaying the loader popup. The markup for that is
```
<div class="loading-backdrop fade-in active" ng-class="{'show-backdrop': showBackdrop}">
    <div style="width: 200px; margin-left: -100px; margin-top: -40px;" class="loading" ng-transclude="">
        <span>
            Loading...
        </span>
    </div>
</div>
```
The bundled CSS file (www/lib/ionic/css/ionic.css) contains rules for .loading-backdrop and .loading, but the CSS generated from SASS (lib/ionic/scss/*) doesn't. Instead it has a rule for a .loading-container. Perhaps outdated rules?

It's quite difficult to compare the generated CSS with the bundled, but if you have any ideas on how to normalize the two into some comparable form, I can give it a try to find more inconsistencies.
